pkg/apis/workflow/v1alpha1/info.go

```diff
@@ -6,7 +6,7 @@ package v1alpha1
 type Link struct {
 	// The name of the link, E.g. "Workflow Logs" or "Pod Logs"
 	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
-	// "workflow", "pod", "pod-logs", "event-source-logs", "sensor-logs" or "chat"
+	// "workflow", "pod", "pod-logs", "event-source-logs", "sensor-logs", "custom-buttons" or "chat"
```

Comment:
Is this used anywhere?

Reply:
Thank you for the review @terrytangyuan!
This `scope`  will only be used by the UI code to create buttons on the workflow view, and it is not used by anywhere for now.
@rbreeze will create UI based on the scope `custom-buttons` in the next UI PRs, and this PR only make the necessary backend change

Reply:
But the docs and manifests are also updated, which might mislead users. Does it make more sense to work on a single PR? @alexec Any thoughts?

Reply:
Personally, I feel like the `main` branch can always contain un-released or incompleted features, and most users use releases and this shouldn't affect them much. But if you think that is an issue, I can also remove the user-facing change(e.g., docs, manifests) in this PR and add them back once the feature is completed.

Reply:
I agree with @jiachengxu - I think it's okay to get this merged first given that it's such a small change to the docs

Reply:
SGTM